docs/examples/ClasspathExample/README.md

```diff
@@ -12,6 +12,3 @@ How to use the loom.config classpath array to specify source folders to compile.
 
 ## Screenshot
 ![ClasspathExample Screenshot](images/screenshot.png)
-
-## Code
-@insert_source
```

Comment:
Moving the source would be pointless because that's the point :) I'll copy-pase the relevant config contents, code is irrelevant here anyway.